### PR TITLE
feat(eslint-ramda): add eslint configuration for Ramda

### PR DESCRIPTION
**Pull request by developer239**
Useful helpers for people who use Ramda in their projects. 🙂

### CODE REVIEW

File: packages/eslint-config-ramda/index.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    'ramda/pipe-simplification': 'error',
+    'ramda/prefer-both-either': 'error',
+    'ramda/prefer-complement': 'error',
+    'ramda/prefer-ramda-boolean': 'error',
+    'ramda/prop-satisfies-simplification': 'error',
+    'ramda/reduce-simplification': 'error',
+    'ramda/reject-simplification': 'error',
+    'ramda/set-simplification': 'error',
+    'ramda/unless-simplification': 'error',
+    'ramda/when-simplification': 'error'
```
Comment (varholak-peter) [inline packages/eslint-config-ramda/index.js:38]:
```suggestion
    'ramda/when-simplification': 'error',
```

Comment (developer239) [reply packages/eslint-config-ramda/index.js]: ✅

**Review by robertrossmann — CHANGES_REQUESTED** [review]
Looks interesting, but after a brief glance through the plugin's docs it seems the rules are exclusively a code style or preference-based, therefore they should all be set to `warn` at most.

Also, please check compatibility with Node.js 12 and ESLint v7 and perhaps see if we can upgrade to their upcoming v3 or the plugin. 🙏 

Also, I will need you to take up the maintenance once this is merged. ❤️ Are you up for it?

**Comment by robertrossmann** [comment]
Closing due to inactivity. Feel free to reopen if you ever find time to push this forward! 👍

**Pull request closed without merging**